homeassistant/components/cookidoo/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: No service actions implemented
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: No service actions implemented
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions:
```

Comment:
still needs to be added there's an include for that, check e.g. tedee

Reply:
removal instructions added, why still relevant?